controllers/fluentpvcbinding_controller.go

```diff
@@ -108,7 +116,7 @@ func (r *fluentPVCBindingReconciler) Reconcile(ctx context.Context, req ctrl.Req
 
 	if !podFound && !b.IsConditionReady() {
```

Comment:
FYI
`Ready` condition is false only until both pod and pvc are found.
https://github.com/st-tech/fluent-pvc-operator/blob/70dcd1589b2436ae2ec26f82f430ba8411a51918/controllers/fluentpvcbinding_controller.go#L151-L153

And, `SetConditionNotReady` is not used in this repository, so the `Ready` condition will never revert to false.